Quaver.Shared/Screens/Edit/Actions/EditorActionManager.cs

```diff
@@ -136,6 +136,11 @@ public class EditorActionManager : IDisposable
         /// </summary>
         public event EventHandler<EditorHitSoundRemovedEventArgs> HitsoundRemoved;
 
+        /// <summary>
+        ///     Event invoked when a custom color has been added to a group of objects
+        /// </summary>
+        public event EventHandler<EditorColorSetEventArgs> ColorSet;
```

Comment:
I might just be nitpicking here, but personally I'd prefer if most references to snap colors explicitly stated that they're about snap colors and not just colors. I think it's mostly because I think about editor layer colors when I read about colors in the context of the editor.